README.md

```diff
@@ -37,7 +39,8 @@ You can also refer to the project's own [Dockerfile.dapper](Dockerfile.dapper) a
 
 ### Makefile
 
-The Makefile should include targets to run everything in Dapper. They're defined in [Makefile.dapper](Makefile.dapper) and can be copied as-is and included. To use Shipyard's target, simply include the [Makefile.inc](Makefile.inc) file in your own Makefile.
+The Makefile should include targets to run everything in Dapper. They're defined in [Makefile.dapper](Makefile.dapper) and can be copied
+as-is and included. To use Shipyard's target, simply include the [Makefile.inc](Makefile.inc) file in your own Makefile.
```

Comment:
```suggestion
The Makefile should include targets to run everything in Dapper.
They're defined in [Makefile.dapper](Makefile.dapper) and can be copied as-is and included.
To use Shipyard's target, simply include the [Makefile.inc](Makefile.inc) file in your own Makefile.
```

Reply:
https://github.com/submariner-io/shipyard/pull/273 is for fixing the violations.